base/number.jl

```diff
@@ -92,6 +92,9 @@ copy(x::Number) = x # some code treats numbers as collection-like
 
 Returns `true` if the value of the sign of `x` is negative, otherwise `false`.
 
+`signbit` is the inverse of `cispi` (which calculates `(-1)^n`):
+`signbit(cispi(b)) == b`. Also, `cispi(n) * abs(n) == n`.
```

Comment:
"`signbit` is the inverse of `cispi`"
Perhaps it would be more accurate with  "a left inverse of `cispi` when `b` is a `Bool`"

`cispi(n) * abs(n) == n`, doesn't seem to hold for e.g., `n=-2`

I'm not sure how helpful this paragraph is for understanding the functionality of `signbit`?

Reply:
I didn't want to use the terminology of "left inverse" since that's quite technical. I want to point to the fact that `signbit` and `cispi` are related. `signbit` itself should be explained elsewhere – but a reminder here might be useful.

Reply:
Calling it inverse (or left inverse) without specifying that it is only valid for `Bool`s seems like a quite puzzling statement. But I guess I mostly think that `cispi` and `signbit` are not that strongly related to motivate adding all this text to the succinct doc string of  `signbit`. If there really is a need to indicate a relation, perhaps a simple ```See also [`cispi`](@ref)``` would be enough?

Reply:
I updated the documentation. I now call them "related" functions.